lib/src/Base/Func/NormInfEnumerateFunction.cxx

```diff
@@ -143,6 +143,16 @@ UnsignedInteger NormInfEnumerateFunction::getMaximumDegreeStrataIndex(const Unsi
   return maximumDegree / getDimension();
 }
 
+/* The marginal enumerate function */
+EnumerateFunctionImplementation NormInfEnumerateFunction::getMarginal(const Indices & indices) const
```

Comment:
```suggestion
EnumerateFunction NormInfEnumerateFunction::getMarginal(const Indices & indices) const
```

Reply:
Thank you very much for the detailed suggestions. Fixed at 8ba0bdcdc775a35dbdf8f2987f7701ee352fa488.